ALUM/ALUM/Views/PreSessionForm/PreSessionConfirmationScreen.swift

```diff
@@ -0,0 +1,136 @@
+                            Text("Summary")
+                                .font(.custom("Metropolis-Regular", size: 34))
+                                .foregroundColor(Color("NeutralGray3"))
+                            Spacer()
+                        }
+                        .padding(.leading, 16)
+                        .padding(.trailing, 16)
+                        .padding(.bottom, 32)
+                        .padding(.top, 8)
+
+                        ForEach(viewModel.questionList, id: \.self) { currQuestion in
+
+                            HStack {
+                                Text(currQuestion.question)
+                                    .foregroundColor(Color("ALUM Dark Blue"))
+                                    .font(Font.custom("Metropolis-Regular", size: 17, relativeTo: .headline))
+
+                                Spacer()
+                            }
+                            .padding(.leading, 16)
+                            .padding(.trailing, 16)
+                            .padding(.bottom, 8)
+
+                            if currQuestion.type == "text" {
+                                HStack {
+                                    Text(currQuestion.answerParagraph)
+                                        .font(Font.custom("Metropolis-Regular", size: 17, relativeTo: .headline))
+
+                                    Spacer()
+                                }
+                                .padding(.leading, 16)
+                                .padding(.trailing, 16)
+                                .padding(.bottom, 32)
+                            } else if currQuestion.type == "bullet" {
+
+                            }
+                        }
+                    }
+                }
+
+                ZStack {
+                    Rectangle()
+                        .frame(maxWidth: .infinity)
+                        .frame(height: 114)
+                        .foregroundColor(Color.white)
+                        .ignoresSafeArea(edges: .all)
+
+                    HStack {
+                        Button {
+                            dismiss()
+                        } label: {
+                            HStack {
+                                Image(systemName: "arrow.left")
+                                Text("Back")
+                            }
+                        }
+                        .buttonStyle(OutlinedButtonStyle(disabled: false))
+                        .padding(.trailing, 16)
+                        .padding(.bottom, 32)
+                        .frame(width: UIScreen.main.bounds.width * 0.3025)
+
+                        Button {
+
+                        } label: {
+                            Text("Save")
+                        }
+                        .buttonStyle(FilledInButtonStyle(disabled: false))
+                        .padding(.bottom, 32)
+                        .frame(width: UIScreen.main.bounds.width * 0.575)
```

Comment:
Let's use `GeometryReader` instead of `UIScreen`. 
1. GeometryReader is SwiftUI's native way of getting size of view
2. `UIScreen.main` is deprecated - https://developer.apple.com/documentation/uikit/uiscreen/1617815-main
3. UIScreen is UIKit's way of getting screen size so less preferred for SwiftUI